autotest/pom.xml

```diff
@@ -98,6 +97,21 @@
             <artifactId>token-client-spring</artifactId>
             <version>${nav.security.version}</version>
         </dependency>
+        <dependency>
+            <groupId>javax.xml.bind</groupId>
+            <artifactId>jaxb-api</artifactId>
+            <version>2.4.0-b180830.0359</version>
+        </dependency>
+        <dependency>
+            <groupId>javax.activation</groupId>
+            <artifactId>activation</artifactId>
+            <version>1.1.1</version>
+        </dependency>
+        <dependency>
+            <groupId>org.glassfish.jaxb</groupId>
```

Comment:
trenger du dette avhengighet?

Reply:
Ja, hvis jeg fjerner den så får jeg feilmelding:
`javax.xml.bind.JAXBException: Implementation of JAXB-API has not been found on module path or classpath.`